src/System.IO.Compression.Brotli/System/IO/Compression/Brotli/BrotliPrimitives.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.CompilerServices;
+using System.Text;
+#if BIT64
+    using nuint = System.UInt64;
+#else
+    using nuint = System.UInt32;
+#endif 
+namespace System.IO.Compression
+{
+    public struct BrotliPrimitives
+    {
+        const int defQuality = 11;
+        const int defLgWin = 24;
+
+        public static bool Compress(ReadOnlySpan<byte> input, Span<byte> output, out nuint consumed, out nuint writen)
```

Comment:
Great to see these APIs!
Would it be possible to have the same signature as Base64 encoder @ahsonkhan just commited: 
https://github.com/dotnet/corefxlab/blob/master/src/System.Binary.Base64/System/Binary/Base64Decoder.cs#L205